lib/intercom/request.rb

```diff
@@ -72,10 +58,16 @@ def execute(target_base_url=nil, username:, secret: nil, read_timeout: 90, open_
         client(base_uri, read_timeout: read_timeout, open_timeout: open_timeout).start do |http|
           begin
             response = http.request(net_http_method)
+
             set_rate_limit_details(response)
-            decoded_body = decode_body(response)
-            parsed_body = parse_body(decoded_body, response)
             raise_errors_on_failure(response)
+
+            parsed_body = extract_response_body(response)
+
+            return nil if parsed_body.nil?
```

Comment:
This line and https://github.com/intercom/intercom-ruby/pull/492/files#diff-e0313851110ec5f90dfd3a2befbc94f3R123 are to prevent any existing tests from breaking, in case users are depending on this behavior.